pulsar-cli-utils/src/main/java/org/apache/pulsar/cli/converters/ByteUnitToLongConverter.java

```diff
@@ -18,22 +18,16 @@
  */
 package org.apache.pulsar.cli.converters;
 
-import static org.apache.pulsar.cli.ValueValidationUtil.emptyCheck;
-import com.beust.jcommander.converters.BaseConverter;
+import picocli.CommandLine;
 
-public class ByteUnitToLongConverter extends BaseConverter<Long> {
-
-    public ByteUnitToLongConverter(String optionName) {
-        super(optionName);
-    }
+public class ByteUnitToLongConverter implements CommandLine.ITypeConverter<Long> {
```

Comment:
You can remove this class, and use `org.apache.pulsar.cli.converters.picocli.ByteUnitToLongConverter` instead.

Reply:
Done.